mbuild/compound.py

```diff
@@ -556,6 +558,13 @@ def charge(self):
         filtered_charges = [charge for charge in charges if charge is not None]
         return sum(filtered_charges) if filtered_charges else None
 
+    @staticmethod
+    def _particle_charge(particle):
+        if particle._charge is not None:
+            return particle._charge
+        else:
+            return None
```

Comment:
I think this statement is redundant. Can change to be:
```python
```suggestion
    def _particle_charge(particle):
        return particle._charge
```